test/suite/test_verbose04.py

```diff
@@ -118,9 +118,9 @@ def test_verbose_all(self):
             session.compact(uri)
             session.close()
 
-        # At this time, no verbose messages should be generated with the following set of operations and the verbosity level
-        # WT_VERBOSE_INFO (0), hence we don't expect any output.
-        with self.expect_verbose(['all:0'], self.all_verbose_categories, self.is_json, False) as conn:
+        # At this time, only INFO verbose messages should be generated with the following set of
+        # operations and the verbosity level WT_VERBOSE_INFO (0).
+        with self.expect_verbose(['all:0'], self.all_verbose_categories, self.is_json) as conn:
```

Comment:
This code change is correct, but it seems like there's a philosophical change here? We've never logged anything at INFO before, although I can see there are now some recent exceptions for compact.

The documentation says INFO is for "describing statuses and actions performed by WiredTiger" so it doesn't seem inappropriate, just a break from tradition.

Reply:
Yes, in the KTLO stream a couple of people raised an opinion on a couple of WT logs to be printed with a default configuration. This is the step towards printing all the WT INFO messages into the MongoDB logs. We plan to add/convert a couple of existing verbose messages into the INFO messages.

Reply:
Cool, that makes sense. Thanks for explaining things.